website/source/api/acl/roles.html.md

```diff
@@ -0,0 +1,462 @@
+---
+layout: api
+page_title: ACL Roles - HTTP API
+sidebar_current: api-acl-roles
+description: |-
+  The /acl/role endpoints manage Consul's ACL Roles.
+---
+
+-> **1.5.0+:**  The APIs are available in Consul versions 1.5.0 and later. The documentation for the legacy ACL API is [here](/api/acl/legacy.html)
+
+# ACL Role HTTP API
+
+The `/acl/role` endpoints [create](#create-a-role), [read](#read-a-role),
+[update](#update-a-role), [list](#list-roles) and [delete](#delete-a-role)  ACL roles in Consul.
+For more information about ACLs, please see the [ACL Guide](/docs/guides/acl.html).
```

Comment:
This link will need to be updated, depending on where we document `roles`. If we decide to add it to the guides then we will need to link out to Learn. However, I thought we wanted to wait on adding roles to the guides since its a bit complicated for users who aren't using IdPs.

Reply:
That stub I left in place when cloning the Policy skeleton for now, but I agree that any links outside of `/api` are subject to change.